Notifier-service/src/services/conference.spec.ts

```diff
@@ -1,16 +1,24 @@
 import { ConferenceModel } from '../interfaces/models/conference'
 import { ConferenceServiceI } from './conference'
+import * as dotenv from 'dotenv'
+import { MongoDb } from '../database/mongodb'
+import { ConferenceModelMongo } from '../models/conference'
+import { ConferenceDocument } from '../schemas/conferences'
 
 describe("Testing Conferences Service Implementation " ,() => {
+    dotenv.config()
```

Comment:
* not required npm test is already including this and metadata reflect.

Reply:
sure, I will remove